lib/services/local/generic/generic.go

```diff
@@ -121,8 +127,13 @@ func (s *Service[T]) WithPrefix(parts ...string) *Service[T] {
 	}
 }
 
+// WithPrefix will return a service with the given parts appended to the backend prefix.
+func (s *service[T]) WithPrefix(parts ...string) Service[T] {
```

Comment:
Same, combine withPrefix and WithPrefix?

Reply:
Removed.